apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloCacheInterceptor.kt

```diff
@@ -185,11 +192,4 @@ class ApolloCacheInterceptor(
       }
     }
   }
-
-  init {
-    this.apolloStore = apolloStore
-    this.dispatcher = dispatcher
-    this.logger = logger
-    this.writeToCacheAsynchronously = writeToCacheAsynchronously
```

Comment:
👍

Reply:
Yeah I guessed, we had some of these as well when we first migrated :)